api/SVGPathElement.json

```diff
@@ -1127,12 +1127,38 @@
               "version_added": "12",
               "version_removed": "79"
             },
-            "firefox": {
-              "version_added": "1.5"
-            },
-            "firefox_android": {
-              "version_added": "4"
-            },
+            "firefox": [
+              {
+                "version_added": "97",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": "dom.svg.pathSeg.enabled",
+                    "value_to_set": "true"
+                  }
+                ]
+              },
+              {
+                "version_added": "1.5",
+                "version_removed": "97"
+              }
+            ],
+            "firefox_android": [
+              {
+                "version_added": "97",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": "dom.svg.pathSeg.enabled",
+                    "value_to_set": "true"
+                  }
+                ]
+              },
```

Comment:
On Android there are no user serviceable parts inside. I think we just need the removal here